python/cugraph/cugraph/utilities/utils.py

```diff
@@ -210,45 +207,42 @@ def get_traversed_path_list(df, id):
     return answer
 
 
-def is_cuda_version_less_than(min_version=(10, 2)):
+def is_cuda_version_less_than(min_version):
```

Comment:
Based on the function name, this should not have a default value. Its default was also outdated.

This function also appears to be unused. Do we want to keep it?